wfc3tools/embedsub.py

```diff
@@ -91,28 +91,43 @@ def embedsub(files):
         flt[1].header['sizaxis1'] = yaxis
         flt[1].header['sizaxis2'] = xaxis
 
-        for i in range(1, 4):
-            flt[i].header['crpix1'] = crpix1 + x1 - 1
-            flt[i].header['crpix2'] = crpix2 + y1 - 1
-            flt[i].header['ltv1'] = 0.0
-            flt[i].header['ltv2'] = 0.0
+        # loop through each possible fits extension and modify keywords if they exist
+        for i in range(1, len(flt)):
+            try:
```

Comment:
instead of a try except here, you could just check 'if crpix1 in flt[i].header.keys'